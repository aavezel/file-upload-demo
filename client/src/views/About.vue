<template>
    <v-container grid-list-xl text-xs-center>
        <v-layout align-center justify-center fill-height>
            <v-flex xs12>
                <h3>Демо проект Александра Коротаева</h3>
                <h4>(с) 2020</h4>
            </v-flex>
        </v-layout>
    </v-container>
</template>
