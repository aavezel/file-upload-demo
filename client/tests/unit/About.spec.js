import Vue from 'vue'
import Vuetify from 'vuetify'
Vue.use(Vuetify)


import About from '@/views/About.vue'

import {
    mount,
    createLocalVue
} from '@vue/test-utils';


const localVue = createLocalVue()


describe('About', () => {

    let vuetify;

    beforeEach(() => {
        vuetify = new Vuetify()
    })

    test('Проверка снапшота', () => {
        const wrapper = mount(About, {
            localVue,
            vuetify
        })

        expect(wrapper.html()).toMatchSnapshot();
        
        const h3 = wrapper.find('h3')
        expect(h3.text()).toBe('Демо проект Александра Коротаева')

        const h4 = wrapper.find('h4')
        expect(h4.text()).toBe('(с) 2020')

    })
})