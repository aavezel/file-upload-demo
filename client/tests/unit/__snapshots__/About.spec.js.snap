// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Проверка снапшота 1`] = `
<div class="container grid-list-xl text-xs-center">
  <div class="layout align-center justify-center fill-height">
    <div class="flex xs12">
      <h3>Демо проект Александра Коротаева</h3>
      <h4>(с) 2020</h4>
    </div>
  </div>
</div>
`;